cmd/flux/create_secret_git.go

```diff
@@ -45,6 +45,12 @@ For Git over HTTP/S, the provided basic authentication credentials are stored in
     --ssh-key-algorithm=ecdsa \
     --ssh-ecdsa-curve=p521
 
+  # Create a Git SSH authentication secret with a private key from file
+  # The public SSH host key will still be gathered from the host
+  flux create secret git podinfo-auth \
+    --url=ssh://git@github.com/stefanprodan/podinfo \
+    --private-key-file=./key
```

Comment:
Is the key a dir? Can we say more about what should be in there?

Reply:
I would use key.pem also mention that it must a key without password?

Reply:
PEM files are OpenSSL public keys, or key chains (and are defined in an RFC), so this wouldn’t be right.

Closest would probable be .key which may contain just a private key.

Note that the key you normally would generate with defaults (still `id_rsa`) is  extensionless.

Reply:
Ok so maybe id_rsa is a better example